### ISSUE TITLE
[Ona Support Request]: Native Form Library for Nepali Language

### ISSUE THREAD

**Issue by uzumakirex0000** [opensrp/opensrp-client-anc#779]
### Affected App or Server Version

v1.6.8

### What kind of support do you need?

We need the Nepali language in native form library !!! 

### What is the acceptance criteria for your support request?

In     .gradle\caches\transforms-2\files-2.1\e7266aa1c0c9ef91d2a30d85e0846994\jetified-opensrp-client-native-form-2.1.11-SNAPSHOT\res\values    we need nepali language as well here because there is still some part where app shows english language.

**Comment by SebaMutuku** [comment]
@uzumakirex0000 please translate the strings.xml shared here https://opensrp.slack.com/archives/C02V60H2KN1/p1644493918397669

**Comment by Naima-Bashir** [comment]
The nepal team to update the XML strings, then Ona can add it to the repo 
@uzumakirex0000

**Comment by Naima-Bashir** [comment]
Instructions for the Nepal team @uzumakirex0000 
1. Clone native form library from github 
2. Pull it from git 
3. Replace values inside java, call it values. Replace strings .xml and create a maven (locally or other)
4. Replace with the edited version both on Opensrp anc and ref app
5.Replace with the translated strings to build.gradle and ANC

Feel free to reach out to the Ona team on slack and we can support

**Comment by SebaMutuku** [comment]
@uzumakirex0000 Have you managed to implement this on your end?

**Comment by uzumakirex0000** [comment]
yes !!! I have cloned the  opensrp-client-native-form and created values_ne and created strings.xml but now I am stuck what should I do next. i need to add something in Client Anc or native form's Build.Gradle ?!!!

**Comment by SebaMutuku** [comment]
Hi @uzumakirex0000 follow these steps for you to publish a local maven build of native form.
You might need to change the version to your desired version in gradle.properties e.g 2.1.14-dev-SNAPSHOT and keep a note of the version you are using currently.
1. open terminal project terminal from your ide
2. run ./gradlew :android-json-form-wizard:uploadArchives -PmavenLocal=true
3. Open WHO ANC application and change native form version on both reference-app and opensrp-anc application to the one that you have published locally. Once you test and ascertained everything is working, push the code,raise a PR. We will review it and publish a new version for you to use in your production application.
4. In case you are stuck on publishing a local release, kindly find attached documentation link https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/65692227/How+to+upload+and+use+module+libraries+on+maven

**Comment by SebaMutuku** [comment]
Hi @uzumakirex0000 . Is there any form of assistance you need from our end?

**Comment by SebaMutuku** [comment]
Hi @Naima-Bashir this issue has been resolved.
